### ISSUE TITLE
[feat]: Fixes in UI of the docs section in dark mode.

### ISSUE THREAD

**Issue by its-me-abhishek** [INCF/artem-is#131]
- Problem: While reviewing documentation, I noticed that hyperlinks became unreadable and unpleasant once I reached the headings.

- Proposed Solution:
  - Resolving the issue can also involve adjusting the color of the -md-typeset-a-color property to a lighter shade.
  - To further enhance the UI, we could consider capitalizing the first letter of texts in the left pane.

- Screenshots :
  - Before:
![prob](https://github.com/INCF/artem-is/assets/114338679/0aa93472-7942-4db0-a031-97201d587333)
  - After:
![sol](https://github.com/INCF/artem-is/assets/114338679/3aa87743-f857-4bab-8497-914bac8a2a4c)

**Comment by Remi-Gau** [comment]
The contrast on the second option seems more readable indeed.

Do you feel like trying to update the mkdocs CSS to improve this?

**Comment by its-me-abhishek** [comment]
Yes. I was thinking of an approach mentioned in mkdocs' docs: https://www.mkdocs.org/user-guide/customizing-your-theme/
We could change the color of the specific tags.

**Comment by Remi-Gau** [comment]
go for it!!

may have to check how only modify how to change link colors only for dark mode but that's CSS magic we can figure out

open a pull request and ping me, I can review it

**Comment by Remi-Gau** [comment]
and don't hesitate to ask if you need help with anything

**Comment by its-me-abhishek** [comment]
Sure! 😄
